crates/ruma-state-res/src/event_auth.rs

```diff
@@ -759,39 +759,6 @@ fn check_redaction<E: Event>(
     Ok(false)
 }
 
-/// Check that the member event matches `state`.
-///
-/// This function returns false instead of failing when deserialization fails.
-fn check_membership<E: Event>(member_event: Option<Arc<E>>, state: MembershipState) -> bool {
-    if let Some(event) = member_event {
-        if let Some(Ok(membership)) = event
-            .content()
-            .get("membership")
-            .map(|m| serde_json::from_value::<MembershipState>(m.clone()))
-        {
-            membership == state
-        } else {
-            false
-        }
-    } else {
-        false
-    }
-}
-
-/// Can this room federate based on its m.room.create event.
-fn can_federate<E: Event>(auth_events: &StateMap<Arc<E>>) -> bool {
```

Comment:
Were `can_federate` and `check_membership` inlined or just not used?

Reply:
I think they were not used at all. You can check whether there's any calls to them in main, which doesn't have any of the refactoring commits.

Reply:
Nope only defined, never used, oops.